common/pom.xml

```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>com.google.firebase</groupId>
       <artifactId>firebase-admin</artifactId>
-      <version>[5.3.0,)</version>
+      <version>[5.9.0,)</version>
```

Comment:
I mean here.

Reply:
Oh yeah, you're right.  I was so used to Android/Gradle land where we constantly bump our dependencies that I just did this instinctively.  I'll revert it.